CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ### Added
 
+- `Dialog`: added `isToggle` prop to leftAction to allow it to be a toggle ([@BeirlaenAaron](https://github.com/BeirlaenAaron)) in [#2409](https://github.com/teamleadercrm/ui/pull/2409))
```

Comment:
Add this point we could just allow for left action to be any custom component we want and have the implementation take care of how it should behave